contracts/cosmwasm-vm/cw-ibc-core/tests/channel/test_timeout.rs

```diff
@@ -224,15 +226,25 @@ fn test_timeout_packet_validate_to_light_client() {
     let _client_type = IbcClientType::new("iconclient".to_string());
 
     let light_client = LightClient::new("lightclient".to_string());
-
     contract
-        .bind_port(&mut deps.storage, &src_port, "moduleaddress".to_string())
+        .store_client_implementations(
+            &mut deps.storage,
+            &IbcClientId::default(),
+            light_client.clone(),
+        )
+        .unwrap();
+    let timestamp_query = light_client
+        .get_timestamp_by_height_query(&IbcClientId::default(), proof_height.revision_height())
         .unwrap();
+    let mut mocks = HashMap::<Binary, Binary>::new();
+    mocks.insert(timestamp_query, to_binary(&0_u64).unwrap());
+    mock_lightclient_query(mocks, &mut deps);
 
     contract
-        .store_client_implementations(&mut deps.storage, &IbcClientId::default(), light_client)
+        .bind_port(&mut deps.storage, &src_port, "moduleaddress".to_string())
         .unwrap();
-    mock_lightclient_reply(&mut deps);
+
+    // mock_lightclient_reply(&mut deps);
```

Comment:
```suggestion
```

Reply:
i have used  mock_lightclient_query(mocks, &mut deps); which mocks both height query and membership query. so its not required anymore.

Reply:
Should then be removed